src/app/hostname-generators/views/HostnameGeneratorDetailView.vue

```diff
@@ -0,0 +1,104 @@
+      name: '',
+    }"
+    v-slot="{ route, t, uri }"
+  >
+    <RouteTitle
+      :title="t('hostname-generators.routes.items.title')"
+      :render="false"
+    />
+
+    <DataLoader
+      :src="uri(sources, '/hostnamegenerators/:name', {
+        name: route.params.name,
+      })"
+      v-slot="{ data }"
+    >
+      <AppView :docs="t('hostname-generators.href.docs')">
+        <template
+          v-if="data"
+          #title
+        >
+          <h1>
+            <XCopyButton
+              :text="data.name"
+            >
+              <RouteTitle
+                :title="t('hostname-generators.routes.item.title', { name: data.name })"
+              />
+            </XCopyButton>
+          </h1>
+        </template>
+        <div class="stack">
+          <AppAboutSection
+            :title="t('hostname-generators.routes.item.subtitle', { name: data.name })"
+            :created="t('common.formats.datetime', { value: Date.parse(data.creationTime) })"
+            :modified="t('common.formats.datetime', { value: Date.parse(data.modificationTime) })"
+          >
+            <div class="columns">
+              <DefinitionCard>
+                <template #title>
+                  {{ t('hostname-generators.common.selector') }}
+                </template>
+
+                <template
+                  v-if="data.selector"
+                  #body
+                >
+                  <XAction
+                    :to="{
+                      name: data.selector?.routeName,
+                      params: {
+                        mesh: data.mesh,
+                      },
+                    }"
+                  >
+                    {{ data.selector?.label }}
+                  </XAction>
+                </template>
+              </DefinitionCard>
+            </div>
+          </AppAboutSection>
+
+          <ResourceCodeBlock
+            v-if="data"
+            :resource="data.$raw"
+            v-slot="{ copy, copying }"
+          >
+            <DataSource
+              v-if="copying"
+              :src="uri(sources, '/hostnamegenerators/:name/as/kubernetes', {
+                name: route.params.name,
+              }, {
+                cacheControl: 'no-store',
+              })"
+              @change="(data) => {
+                copy((resolve) => resolve(data))
+              }"
+              @error="(e) => {
+                copy((_resolve, reject) => reject(e))
+              }"
+            />
+          </ResourceCodeBlock>
+        </div>
+      </AppView>
+    </DataLoader>
+  </RouteView>
+</template>
+
+<script lang="ts" setup>
+import { AppAboutSection } from '@kong-ui-public/app-layout'
+
+import { sources } from '../sources'
+import DefinitionCard from '@/app/common/DefinitionCard.vue'
+import ResourceCodeBlock from '@/app/x/components/x-code-block/ResourceCodeBlock.vue'
+</script>
+
+<style lang="scss" scoped>
+:deep(.kong-ui-app-about-section .about-section-content) {
+  display: block;
+}
```

Comment:
Do you know what this was for? (not sure if I did this somewhere else and its just copy/pasta'ed or whether you needed to add this for something else)

Reply:
I copied this over from another place that uses `AppAboutSection`. Thought it's required to display the inner content properly 🤔

Reply:
So I've checked this again and with just one item in the view this can be omitted. As soon as we add more stuff to the `AppAboutSection` the layout is better with that `display: block` on the element.